### DISCUSSION TITLE
Sorting table in front-end composer

### DISCUSSION THREAD

**Discussion by rmuller-vu** [mia-platform/community#550] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

I am wondering if there is a way to group sorts when using the table component in the frontend composer. 

URL: `mfe-application/version-approval?pageSize=10&pageNumber=1&characteristic=tab-1&sortProperty=approvedOn&sortProperty=deniedOn&sortProperty=versionCreatedAt&sortDirection=ascend`

I would like to sort `approvedOn` and `deniedOn` with a `sortDirection` of ascend. I would also like to sort `versionCreatedAt` by descend. Is this possible.

My scenario is `approvedOn` and `deniedOn` are datetime fields that are null when they have no value. I would like the rows with both `approvedOn` and `deniedOn` that have the value null to be on top but then to have them sorted by newest to oldest by the `versionCreatedAt` datetime field.

**Comment by LiliumCandidum** [comment]
Hi @rmuller-vu,

Sadly, it's not possible to sort more than one column at a time and the only thing I can suggest you is to do it in the backend.

**Reply by rmuller-vu** [reply, marked as answer]
Thank you for the confirmation
